osu.Game/Overlays/Rankings/Tables/RankingsTable.cs

```diff
@@ -84,31 +86,36 @@ private void load()
         {
             AutoSizeAxes = Axes.Both,
             Direction = FillDirection.Horizontal,
-            Spacing = new Vector2(7, 0),
+            Spacing = new Vector2(10, 0),
+            Margin = new MarginPadding { Bottom = row_spacing },
             Children = new[]
             {
                 new UpdateableFlag(GetCountry(item))
                 {
-                    Size = new Vector2(20, 13),
+                    Size = new Vector2(30, 20),
                     ShowPlaceholderOnNull = false,
                 },
                 CreateFlagContent(item)
             }
         };
 
+        protected virtual IEnumerable<string> GradeColumns() => new List<string>();
+
         protected virtual string HighlightedColumn() => @"Performance";
 
         private class HeaderText : OsuSpriteText
         {
             private readonly string highlighted;
 
-            public HeaderText(string text, string highlighted)
+            public HeaderText(string text, string highlighted, IEnumerable<string> gradeColumns)
             {
                 this.highlighted = highlighted;
 
                 Text = text;
                 Font = OsuFont.GetFont(size: 12);
-                Margin = new MarginPadding { Horizontal = 10 };
+
+                var isGrade = gradeColumns.Contains(text);
+                Margin = new MarginPadding { Vertical = 5, Horizontal = isGrade ? 20 : 10 };
```

Comment:
The header text should know nothing about grade columns. Set the margin higher-up explicitly however you need it.

Reply:
Done, also changed the highlight logic a bit to only pass a bool instead of the whole highlighted column string.